styles.css

```diff
@@ -0,0 +1,164 @@
+@media only screen and (max-width: 600px) {
+  #navbar div {
+    display: flex;
+    gap: 0.5em;
+    padding: 0.5vw 1vw;
+  }
+
+  nav .flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .flex-container a {
+    margin: 0;
+    text-align: left;
+  }
+
+  #navbar div a {
+    text-align: center;
+  }
+
+  #navbar header {
+    margin: auto;
+    padding: 0;
+  }
+}
+
+@media only screen and (min-width: 600.1px) {
+  #navbar {
+    float: left;
+    margin: 1em;
+    padding: 1em;
+    width: 15vw;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    position: fixed;
+    background-color: #a52a2a;
+  }
+
+  #main-doc {
+    float: right;
+    width: 70vw;
+    border-left: 3px solid #000;
+    padding-left: 2vw;
+    padding-right: 2vw;
+    background-color: #f5f5dc;
+  }
+
+  #navbar a {
+    display: block;
+    text-align: left;
+    width: 100%;
+    padding: 1vw 0;
+    font-size: 1.25rem;
+    text-decoration: none;
+    filter: opacity(90%);
+    color: burlywood;
+  }
+
+  section .flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  div .nav-link:first-child {
+    padding-top: -5em;
+  }
+}
+
+body {
+  font-family: 'Times New Roman', Times, serif;
+  background-color: #a52a2a;
+  color: #050303;
+  width: 100%;
+  margin: auto;
+  text-align: justify;
+}
+
+/* Navigation bar */
+#navbar {
+  padding-left: 1vw;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+#main-doc {
+  padding-left: 2vw;
+  padding-right: 2vw;
+  background-color: #f5f5dc;
+}
+
+.main-section header {
+  font-size: xx-large;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: bolder;
+  font-style: italic;
+  text-transform: uppercase;
+  text-align: left;
+  filter: blur(1px);
+}
+
+#navbar header {
+  font-size: xx-large;
+  border-top: 0.5vw solid #a52a2a;
+  border-bottom: 0.5vw solid #a52a2a;
+  width: max-content;
+  background-color: #a52a2a;
+  color: #deb887;
+}
+
+#navbar a {
+  font-size: 1.25rem;
+  text-decoration: none;
+  filter: opacity(90%);
+  border-top: 0.5vw solid #a52a2a;
+  border-bottom: 0.5vw solid #a52a2a;
+  background-color: #a52a2a;
+  color: #deb887;
+}
+
+#navbar header:hover,
+#navbar a:hover {
+  color: #a52a2a;
+  background-color: #deb887;
+  filter: opacity(100%);
+  border-radius: 1em;
+  transition: 0.5s ease;
+}
+
+#navbar a:first-child {
+  padding-top: 5vw;
+}
+
+/* Main element */
+.main-section {
+  border-bottom: 0.5vw solid #000;
+  padding: 1vw;
+}
+
+/* Section element */
+.flex-container li {
+  margin: 10px;
+  text-align: left;
+}
+
+main section:first-child,
+main section:last-child {
+  display: block;
+  text-align: center;
+}
+
+main section:first-child {
+  padding-top: 4vw;
+  padding-bottom: 3vw;
```

Comment:
Same here, please use the short form of the padding attribute.

Reply:
As earlier highlighted, it's best practice to use the shorthand property. Please use the short form of the padding attribute.